### ISSUE TITLE
Problem with multiple User-agent lines

### ISSUE THREAD

**Issue by hbea2014** [tomverran/robots#3]
I tested your class with multiple User-agent lines and I got some unexpected results:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

use tomverran\Robot\RobotsTxt;

$robotsTxt = new RobotsTxt('
User-agent: *
User-agent: UA
User-agent: Googlebot
Allow: /
Disallow: /private/
Disallow: /secret/page.html
');

var_dump(
    $robotsTxt->isAllowed('*', '/private/page.html'), // returns true instead of false
    $robotsTxt->isAllowed('UA', '/private/page.html'), // returns true instead of false
    $robotsTxt->isAllowed('Googlebot', '/private/page.html') // returns true instead of false
);
```

**Comment by cristiansantana** [comment]
first, i don't speak english... so.. sorry ;)

your example has a bad sintax (really not bad syntax... maybe... bad order for rules). The first rule than match will be applied, so '/private/page' first match with 'Allow: /'...

**Comment by tomverran** [comment]
Thanks for reporting these issues! Issues should be fixed in 1.02-beta. I will get some more tests in place before marking it as production ready, Re-opening this issue so I don't forget

**Comment by tomverran** [comment]
I'm confident this now works so finally closing this

Thanks again for raising the issue!
